pkg/component/worker/kubelet.go

```diff
@@ -170,15 +191,21 @@ func (k *Kubelet) Start(ctx context.Context) error {
 		args["--node-labels"] = strings.Join(k.Labels, ",")
 	}
 
+	nodename, err := node.GetNodename("")
+	if err != nil {
+		return fmt.Errorf("can't get hostname: %v", err)
+	}
```

Comment:
Any reason this has moved out of the Windows specific branch? It's only used there. Moreover, this regressed to use `%v` again. Maybe some remnant of a conflict resolution?

Reply:
I use the the nodename directly after: `getIPs(nodename)`. Since I need the nodename to get the IPs, i avoid calling `node.GetNodename` twice by moving out the windows specific branch.